scripts/open_failed.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+from docopt import docopt
+import logging
+import re
+import os
+import sys
+
+doc = """
+usage: 
+	teuthology-open-failed -h --help
+	teuthology-open-failed <job_name>
+
+Options:
+	-h, --help            show this help message and exit
+	-v					  Shows 
+	<job_name>              The job_id of the job to teuthology logs of failed
+						  jobspec's
+
+opens all failed jobs using scrape.log in vim editor:
+from job name, finds all the failed and dead job and opens them in vim tabs.
+
+NOTE: Must be run on the same machine that is executing the teuthology job
+processes.
+
+"""
+
+def open_failed(job_name):
+	archive = "/ceph/teuthology-archive/" + job_name + "/"
+
+	scrape = archive + "scrape.log"
```

Comment:
I suggest to use report module to get job status, because sometimes the one want to open logs for still running suite and scrape.log just does not exist.